tests/test_server.py

```diff
@@ -406,43 +406,6 @@ async def test_resource_action_update(server_multi, client_multi, environment_mu
     assert result.result["metadata"]["deploy_summary"]["by_state"]["deployed"] == 2
 
 
-async def test_get_environment(client, clienthelper, server, environment):
```

Comment:
I assume we are already testing `environment_get` somewhere else, correct ?

Reply:
Yes, on test_project.py